curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/configure-helmet-using-the-parent-helmet-middleware.english.md

```diff
@@ -8,19 +8,21 @@ challengeType: 2
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-infosec/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-infosec/'>GitHub</a>.
 app.use(helmet()) will automatically include all the middleware introduced above, except noCache(), and contentSecurityPolicy(), but these can be enabled if necessary. You can also disable or configure any other middleware individually, using a configuration object.
-// Example
-<code>app.use(helmet({</code>
-<code>  frameguard: {         // configure</code>
-<code>    action: 'deny'</code>
-<code>  },</code>
-<code>  contentSecurityPolicy: {    // enable and configure</code>
-<code>   directives: {</code>
-<code>     defaultSrc: ["self"],</code>
-<code>     styleSrc: ['style.com'],</code>
-<code>   }</code>
-<code>  },</code>
-<code> dnsPrefetchControl: false     // disable</code>
-<code>}))</code>
+<h3>Example:</h3>
+<blockquote>
+app.use(helmet({<br>
+&nbsp;frameguard: {         // configure<br>
```

Comment:
The blockquote's should use two non breaking spaces for each level of indentation....
```
&nbsp&nbsp;frameguard: {         // configure<br>
&nbsp;&nbsp;&nbsp;&nbsp;action: 'deny'<br>
...
```